src/common/mailing.ts

```diff
@@ -69,36 +75,47 @@ export const resubscribeUser = async (
 };
 
 export const syncSubscription = async function (
-  userId: string,
+  userIds: string[],
   con: DataSource,
 ): Promise<void> {
   if (!process.env.CIO_APP_KEY) {
     return;
   }
 
-  const atts = await cioApi.getAttributes(userId);
-  const subs = JSON.parse(
-    atts?.customer?.attributes?.cio_subscription_preferences || '{}',
+  // This is fetched from APIClient source code
+  // TODO: Remove this once APIClient supports fetching attributes in bulk
+  const request = new CIORequest(process.env.CIO_APP_KEY, {});
+  const userAttributes = await request.post(
+    `${RegionUS.apiUrl}/customers/attributes`,
+    { ids: userIds },
   );
```

Comment:
The function didn't exist in their APIClient, so I extracted the same way they do it internally.
If it is alright with @rebelchris, I'll open a PR with CIO to add it. Probably after I'm done with the notifications endpoint, just to get that out of the way first.

Reply:
Happy with that :) 
Nice work on fixing it for them, keep us posted 🙏